src/server/handler/notificationHandler.ts

```diff
@@ -16,16 +16,18 @@ export default class NotificationHandler extends Handler {
 
     private async getAll(req: express.Request, res: express.Response): Promise<void> {
         try {
-            const notifications = await Notification.find({userId: req.query.token}).populate("subscriber");
+            // @ts-ignore
```

Comment:
why is this here? consider using https://github.com/hopperteam/hopper-server/blob/master/src/handler/authMiddleware.ts#L10-L16